react-native-purchases-ui/android/gradle.properties

```diff
@@ -0,0 +1,5 @@
+PurchasesUi_kotlinVersion=1.7.21
```

Comment:
Hmm I wonder if we should take the chance to update this... But if it's going to work in the same android project as the main sdk, that might not be possible, so I'm ok with this.

Reply:
both will be imported together as modules, so they will be in the same project right?

Reply:
Yup true, nevermind then